jenkinsfile-runner-base-image/packager-config.yml

```diff
@@ -15,8 +15,11 @@ buildSettings:
         #TODO: replace once Jenkinsfile Runner archives runner & CWP is updated
         noCache: true
       source:
-        git: https://github.com/jenkinsci/jenkinsfile-runner.git
-        branch: "1.0-beta-19"
+        # Use fork until fixes are available in upstream
+        # git: https://github.com/jenkinsci/jenkinsfile-runner.git
+        # branch: "1.0-beta-19"
+        git: https://github.com/SAP/stewardci-jenkinsfilerunner-image
+        branch: "jenkinsfile-runner--1.0-beta-19-steward-1"
```

Comment:
As far as I can see this branch has not the correct content.
Only branch jenkinsfile-runner--1.0-beta-19-steward has similar content as 
https://github.com/jenkinsci/jenkinsfile-runner.git

Reply:
The Git repo is also wrong :-(

Reply:
Actually there should be only a tag `jenkinsfile-runner--1.0-beta-19-steward`.

But I don't know how a branch with the same name but different content has been created. I've deleted the branch.